qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/connector/PlacementConnectorImpl.kt

```diff
@@ -4,15 +4,21 @@ import com.google.gson.JsonObject
 import com.qubit.android.sdk.api.placement.PlacementMode
 import com.qubit.android.sdk.api.placement.PlacementPreviewOptions
 import com.qubit.android.sdk.internal.common.logging.QBLogger
+import com.qubit.android.sdk.internal.configuration.repository.ConfigurationModel
+import com.qubit.android.sdk.internal.configuration.repository.ConfigurationRepository
 import com.qubit.android.sdk.internal.placement.model.PlacementModel
+import okhttp3.OkHttpClient
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import java.io.IOException
+import java.util.concurrent.TimeUnit
 
-internal class PlacementConnectorImpl : PlacementConnector {
+internal class PlacementConnectorImpl(
+    configurationRepository: ConfigurationRepository
+) : PlacementConnector {
 
   companion object {
     private const val PLACEMENT_GRAPH_QL_QUERY = "query PlacementContent(\n" +
```

Comment:
BTW It is better to use """ for such long multiline string literals.

Reply:
Right, fixed